### ISSUE TITLE
1.4.1 incompatible with Jellyfin 10.8 beta 1

### ISSUE THREAD

**Issue by avion540** [firecore/InfuseSync#12]
Error log:
```
[ERR] [1] Emby.Server.Implementations.Plugins.PluginManager: Failed to load assembly /config/plugins/InfuseSync_1.4.1.0/InfuseSync.dll. This error occurs when a plugin references an incompatible version of one of the shared libraries. Disabling plugin.
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Method 'ExecuteAsync' in type 'InfuseSync.ScheduledTasks.HousekeepingTask' from assembly 'InfuseSync, Version=1.4.1.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
Could not load type 'SQLitePCL.pretty.IResultSetValue' from assembly 'SQLitePCL.pretty, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.Assembly.GetTypes()
   at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
System.TypeLoadException: Method 'ExecuteAsync' in type 'InfuseSync.ScheduledTasks.HousekeepingTask' from assembly 'InfuseSync, Version=1.4.1.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

**Comment by anthonylavado** [comment]
Thanks for filing this. I have also let Firecore know a few days ago, so hopefully we'll see it fixed soon.

**Comment by avion540** [comment]
Fixed with 6f6f75c66b021c96c091895928659db9f44c6dbb
